src/Components/AllArticles.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import getAllArticles from "../../utils/get-all-articles";
+
+function AllArticles() {
+  const [allArticles, setAllArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((articles) => {
+      setAllArticles(articles);
+    });
+  }, []);
+
+  return (
+    <div className="homePageArticles">
+      <h2 className="articleHeader">Articles</h2>
+      <ul className="allArticles">
+        {allArticles.map((article) => {
+          return (
+            <li className="listItems" key={article.id}>
+              <img src={article.article_img_url} className="img" />
+              <br />
+              <b>{article.title}</b>
+              <br />
+              {article.author}
+              <br />
+              Topic: {article.topic}
+              <br />
+              Comments: {article.comment_count}
+              <br />
+              Votes: {article.votes}
+              <br />
+              Created at: {article.created_at}
+              <br />
+            </li>
```

Comment:
This is a lot of loose HTML code in your `AllArticles` component, perhaps the code for each `<li>` and it's contents could be extracted to another Reach component and imported into here ? 💡